pages/k8s/cdk-addons.md

```diff
@@ -63,10 +63,12 @@ juju config kubernetes-master enable-dashboard-addons=false
 juju config kubernetes-master enable-dashboard-addons=true
 ```
 
-For additional control over the Kubernetes Dashboard you can also deploy it into the cluster
-using the [Kubernetes Dashboard operator bundle](kubernetes-dashboard-bundle). To do so, set
-the `enable-dashboard-addon` [kubernetes-master configuration][] option to `false` and
-deploy the charm into a Kubernetes model on your cluster:
+For additional control over the Kubernetes Dashboard (Different versions,
+authentication methods...) you can also deploy it into the cluster using the
+[Kubernetes Dashboard operator bundle](kubernetes-dashboard-bundle).
+
+To do so, set the `enable-dashboard-addon` [kubernetes-master configuration][]
```

Comment:
that config parameter is still 'enable-dashboard-addons' ;)

Reply:
Sorry about that! Should be fixed now.